app/Http/Controllers/UserController.php

```diff
@@ -36,9 +45,20 @@ public function create()
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(UserCreateRequest $request)
     {
-        //
+        try {
+            $user = $this->userRepository->store($request);
+            $data = ['name' => $request->name, 'email' => $request->email, 'password' => $request->password];
+            $email = $request->email;
+            Mail::send('mail/create_user_mail', $data, function ($message) use ($email) {
+                $message->to($email)->subject('Create successful account');
```

Comment:
Đổi subject này luôn cho hay hơn nhé  "Welcome to Chatwork Forwarder Application"